src/lib/cameraControls.ts

```diff
@@ -32,19 +32,19 @@ export function mouseControlsToCameraSystem(
   mouseControl: MouseControlType | undefined
 ): CameraSystem | undefined {
   switch (mouseControl) {
-    case 'kitty_cad':
+    case 'kittycad':
       return 'KittyCAD'
-    case 'on_shape':
+    case 'onshape':
       return 'OnShape'
-    case 'trackpad_friendly':
+    case 'trackpadfriendly':
       return 'Trackpad Friendly'
     case 'solidworks':
       return 'Solidworks'
     case 'nx':
       return 'NX'
     case 'creo':
       return 'Creo'
-    case 'auto_cad':
+    case 'autocad':
```

Comment:
Not entirely sure what was up there, but setting `lowercase` in rust made it just more consistent it seems across the PascalCase stuff like KittyCad or OnShape that weren't working and the Title Case stuff like Trackpad Friendly that was working.

Reply:
Should add a test for this at `e2e/playwright/testing-settings.spec.ts`

Reply:
This is because `AutoCAD` isn't camel case. `AutoCad` is. `XMLHttpRequest` is what we never want.

I would definitely check that whatever we do, it doesn't break on people's existing settings.

Reply:
You're right. In this case we need to make the TS side work with these original snake_case keys then

Reply:
Well, until we understand it, this is better than being broken.

Reply:
But `tsc` still isn't happy.

Reply:
Shoot I messed up, let me fix it

Reply:
Updated, passes tsc and fmt-check locally now